### DISCUSSION TITLE
OpenTSDB data source should allow variable in aggregation field

### DISCUSSION THREAD

**Discussion by ugtar** [grafana/grafana#71235] [Data Sources]
The opentsdb data source currently has drop-downs for aggregations for both metric aggregation and down sampling. These are pre-populated with the aggregation methods supported by opentsdb.
![image](https://github.com/grafana/grafana/assets/96333/9189be23-e246-4167-9bff-895c7dfc0449)
The Aggregation field should allow one to enter a variable so that the aggregation method can be changed by the dashboard viewer

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82602. Feel free to continue the discussion around this there. Thank you!
